charts/fluentbit/README.md

```diff
@@ -0,0 +1,20 @@
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.  Please refer to
+Helm's [documentation](https://helm.sh/docs) to get started.
+
+Once Helm has been set up correctly, add the repo as follows:
+
+helm repo add calyptia https://calyptia.github.io/charts
+
+If you had already added this repo earlier, run `helm repo update` to retrieve
+the latest versions of the packages.  You can then run `helm search repo
+<alias>` to see the charts.
+
```

Comment:
For AWS marketplace we need to set up a service account with the correct IAM roles:

```shell
$ eksctl create cluster
$ eksctl utils associate-iam-oidc-provider --cluster fabulous-rainbow-1677179292 --approve
$ eksctl create iamserviceaccount --name register-usage-service-account --namespace default --cluster fabulous-rainbow-1677179292 --attach-policy-arn arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage --approve
```